src/api/app/controllers/webui/package_controller.rb

```diff
@@ -701,9 +701,10 @@ def save_file
           errors << 'No file or URI given'
         end
       end
-
-    rescue APIException, StandardError => exception
-      errors << exception.message
+    rescue ActiveXML::Transport::Error => e
```

Comment:
Network unreachable would probably be a http timeout error. How would you do it? @hennevogel